translator/translate/otel/pipeline/prometheus/translators.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package prometheus
+
+import (
+	"go.opentelemetry.io/collector/confmap"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/pipeline"
+)
+
+func NewTranslators(conf *confmap.Conf) pipeline.TranslatorMap {
+	translators := common.NewTranslatorMap[*common.ComponentTranslators]()
+	var destinations []string
+	if conf.IsSet(LogsKey) {
+		destinations = append(destinations, common.CloudWatchLogsKey)
+	}
+	if conf.IsSet(MetricsKey) && conf.IsSet(common.ConfigKey(common.MetricsKey, common.MetricsDestinationsKey, common.AMPKey)) {
```

Comment:
Don't check to see if the AMP key is there. We want to set AMP as a destination so long as the `metrics/metrics_collected/prometheus` section exists.

Reply:
That's invalid config anyways. I think we should check so there is no need to run additional steps by going into translator logic.